src/handlers/payout/action.ts

```diff
@@ -189,7 +189,14 @@ export const handleIssueClosed = async () => {
   logger.info(`Posting a payout url to the issue, url: ${payoutUrl}`);
   const comment =
     `#### Task Assignee Reward\n### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
-  const permitComments = comments.filter((content) => content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot);
+  const permitComments = comments.filter((content) => {
+    if (content.body.includes("https://pay.ubq.fi") && content.user.type == UserType.Bot) {
```

Comment:
use `permitBaseUrl` from the config and extract this into helper function and use it every where we search for permits

Reply:
@whilefoo Hi, please confirm what should be done here

Reply:
No you don't need to add `?claim`